packages/vcr-test-viewer/src/App.tsx

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+
+import { BrowserAuthorizationClientConfiguration } from "@bentley/frontend-authorization-client";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { Viewer } from "@itwin/web-viewer-react";
+import React, { useEffect, useState } from "react";
+import "./App.scss";
+import { Header } from "./Header";
+import { history } from "./history";
+
+const App: React.FC = () => {
+  const [isAuthorized, setIsAuthorized] = useState(
+    (IModelApp.authorizationClient?.hasSignedIn &&
+      IModelApp.authorizationClient?.isAuthorized) ||
+      false
+  );
+  const [isLoggingIn, setIsLoggingIn] = useState(false);
+  const [iModelId, setIModelId] = useState(process.env.IMJS_IMODEL_ID);
+  const [buddiRegion, ] = useState(process.env.IMJS_BUDDI_REGION ? Number(process.env.IMJS_BUDDI_REGION) : 102);
```

Comment:
I'd want the default to be `0` instead of defaulting to our internal environment.

Reply:
agreed, just used this for testing purposes. will be removed when pr gets published and ready to get merged